src/utils/vp.ts

```diff
@@ -0,0 +1,218 @@
+import { formatBytes32String } from '@ethersproject/strings';
+import { getAddress } from '@ethersproject/address';
+import {
+  getProvider,
+  getSnapshots,
+  Multicaller,
+  subgraphRequest
+} from '../utils';
+import _strategies from '../strategies';
+import subgraphs from './subgraphs.json';
```

Comment:
We can use this instead of separate JSON? https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/utils.ts#L93

Reply:
We need a list of subgraph which support delegation, not just the list of all subgraphs

Reply:
@bonustrack These are just delegation subgraphs same as https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L18-L28 (Which are reused in snapshot UI) we moved them to snapshot.js to maintain these URLs from a single place,